.eslintrc.js

```diff
@@ -1,3 +1,22 @@
+const typeScriptModuleOverrides = path => ({
+  files: [`${path}/**/*.ts`],
+  parser: '@typescript-eslint/parser',
+  excludedFiles: ['**/dist/**'],
+  parserOptions: {
+    project: `${path}/tsconfig.json`,
+  },
+  rules: {
+    '@typescript-eslint/camelcase': 'off',
```

Comment:
Why do we need this?

Reply:
We don't - I misread that as `deny-camelcase`, removing...